sqlmesh/core/model/definition.py

```diff
@@ -2236,6 +2249,37 @@ def _list_of_calls_to_exp(value: t.List[t.Tuple[str, t.Dict[str, t.Any]]]) -> ex
     )
 
 
+def _extract_audit_expressions(
+    audits: t.Optional[dict[str, Audit]] = None,
+    inline_audits: t.Optional[dict[str, Audit]] = None,
+    model_audits: t.Optional[t.Any] = None,
+    default_audits: t.Optional[t.List[AuditReference]] = None,
+) -> list:
+    audit_names = []
+    if model_audits:
+        if isinstance(model_audits, (exp.Tuple, exp.Array)):
+            audit_names = [extract_audit(i)[0] for i in model_audits.expressions]
+        elif isinstance(model_audits, exp.Paren):
+            audit_names = [extract_audit(model_audits.this)[0]]
+        elif isinstance(model_audits, exp.Expression):
+            audit_names = [extract_audit(model_audits)[0]]
+
+    if default_audits:
+        for audit_name, _ in default_audits:
+            audit_names.append(audit_name)
+
+    audit_expressions = []
+    if audit_names and audits:
+        for audit_name in audit_names:
+            if audit_name in audits:
+                audit_expressions.append(audits[audit_name].query)
```

Comment:
```suggestion
            audit = audits.get(audit_name)
            if audit:
                audit_expressions.append(audit.query)
```

Reply:
Thanks refactored this and added test for default audits